src/components/App.js

```diff
@@ -1,10 +1,20 @@
 import React from 'react'
+import EmptyEditor from '../components/Editor/EmptyEditor'
```

Comment:
`Empty` suggests a state rather than the functionality of the component.  
I suggest you have `Editor` as your component name and `isEmpty` as a state property

Reply:
I will do as you say.